SampleProject/VOKCoreDataManagerTests/VOKManagedObjectAdditions.m

```diff
@@ -156,17 +154,17 @@ - (void)testRecordInsertionBackgroundThreadConvenienceReturningManagedObjectsArr
                                  @"numberOfHats" : @1,
                                  },
                              ];
-
+    
     [VOKThing vok_addWithArrayInBackground:importArray
-                               completion:^(NSArray *arrayOfManagedObjects) {
-                                   XCTAssertEqual(arrayOfManagedObjects.count, importArray.count);
-                                   for (NSInteger i = 0; i < importArray.count; i++) {
-                                       XCTAssertEqualObjects([arrayOfManagedObjects[i] name], [importArray[i] valueForKey:@"name"]);
-                                       XCTAssertEqualObjects([arrayOfManagedObjects[i] numberOfHats], [importArray[i] valueForKey:@"numberOfHats"]);
-                                   }
-                                   [completionHandlerExpectation fulfill];
-    }];
-
+                                completion:^(NSArray *arrayOfManagedObjects) {
+                                    XCTAssertEqual(arrayOfManagedObjects.count, importArray.count);
+                                    for (NSInteger i = 0; i < importArray.count; i++) {
+                                        XCTAssertEqualObjects([arrayOfManagedObjects[i] name], [importArray[i] valueForKey:@"name"]);
+                                        XCTAssertEqualObjects([arrayOfManagedObjects[i] numberOfHats], [importArray[i] valueForKey:@"numberOfHats"]);
```

Comment:
Actually, it should be using subscript syntax: `importArray` is an array of dictionaries.